docs/source/dataset_summary.rst

```diff
@@ -47,7 +47,13 @@ P300/ERP
    BNCI2014008, 8, 8, 3500 NT / 700 T, 1s, 256Hz, 1
    BNCI2014009, 10, 16, 1440 NT / 288 T, 0.8s, 256Hz, 3
    BNCI2015003, 10, 8, 1500 NT / 300 T, 0.8s, 256Hz, 1
-   bi2013a, 24, 16, 3200 NT / 640 T, 1s, 512Hz, 8
+   bi2012, 25, 16, 6140 NT / 128 T, 1s, 512Hz, 2
+   bi2013a, 24, 16, 3200 NT / 640 T, 1s, 512Hz, 8 for subjects 1-7, else 1
+   bi2014a, 71, 16, , 1s, 512Hz, up to 3
+   bi2014b, 38, 32, , 1s, 512Hz, 3
+   bi2015a, 50, 32, , 1s, 512Hz, 3
+   bi2015b, 44, 32, , 1s, 512Hz, 2
+   VirtualReality, 24, 16, 600 NT / 120 T, 1s, 512Hz, 2
    EPFLP300, 8, 32, 2753 NT / 551 T, 1s, 2048Hz, 4
    Lee2019_ERP, 54, 62, 6900 NT / 1380 T, 1s, 1000Hz, 2
 
```

Comment:
Thank you so much for this! =)

Reply:
@sylvchev, with this, we will not include four datasets in the dataset summary, was it intentional? I know that with DemonP300 and Sosulski2019 dataset it was, but the other two I don't know.

DemonsP300()
Huebner2017([interval, raw_slice_offset, ...])
Huebner2018([interval, raw_slice_offset, ...])
Sosulski2019([use_soas_as_sessions, ...])

Reply:
Hi @bruAristimunha Thank you very much for the suggestions!
I saw the dataset summary was missing for bi2012, bi2014a/b and, bi2015a/b, so I completed it with the information I have.

Reply:
@bruAristimunha we could add the Huebner and Sosulski dataset, we should not include the Demon one

Reply:
Okay @sylvchev!